src/routes/voteOnSponsorTime.js

```diff
@@ -104,15 +108,18 @@ module.exports = async function voteOnSponsorTime(req, res) {
                       json: {
                           "embeds": [{
                               "title": data.items[0].snippet.title,
-                              "url": "https://www.youtube.com/watch?v=" + submissionInfoRow.videoID + 
-                                          "&t=" + (submissionInfoRow.startTime.toFixed(0) - 2),
-                              "description": "**" + row.votes + " Votes Prior | " + (row.votes + incrementAmount - oldIncrementAmount) + " Votes Now | " + row.views + 
-                                              " Views**\n\nSubmission ID: " + UUID + 
-                                  "\n\nSubmitted by: " + submissionInfoRow.userID + "\n\nTimestamp: " + 
+                              "url": "https://www.youtube.com/watch?v=" + submissionInfoRow.videoID 
+                                  + "&t=" + (submissionInfoRow.startTime.toFixed(0) - 2),
+                              "description": "**" + row.votes + " Votes Prior | " + (row.votes + incrementAmount - oldIncrementAmount) + " Votes Now | " + row.views 
+                                  + " Views**\n\n**Submission ID:** " + UUID 
+                                  + "\n\n**Submitted by:** "+submissionInfoRow.userName+"\n " + submissionInfoRow.userID 
+                                  + "\n\n**Total User Submissions:** "+submissionInfoRow.count
+                                  + "\n**Disregarded User Submissions:** "+submissionInfoRow.disregarded
+                                  +"\n\n**Timestamp:** " + 
                                       getFormattedTime(submissionInfoRow.startTime) + " to " + getFormattedTime(submissionInfoRow.endTime),
                               "color": 10813440,
                               "author": {
-                                  "name": userSubmissionCountRow.submissionCount === 0 ? "Report by New User" : (vipRow.userCount !== 0 ? "Report by VIP User" : "")
+                                  "name": userSubmissionCountRow.submissionCount === 0 ? "Report by New User" : (vipRow.userCount !== 0 && "Report by VIP User")
```

Comment:
Won't this display "false"?

Reply:
you're right i forgot to move the close bracket

Reply:
and the fact i didn't even need to change it